### ISSUE TITLE
Map - Onclick

### ISSUE THREAD

**Issue by Code4SocialGood** [Code4SocialGood/c4sg-web#1569]
Country Count:
Org Count: 

![image](https://user-images.githubusercontent.com/24813620/32644612-4c9e9924-c5a8-11e7-8f41-ee34962e5aaf.png)

**Comment by dmgardiner25** [comment]
Are you just wanting the keywords search bar to be populated by the country code automatically? If so, searching by country code can give users from other countries (https://gyazo.com/a7a09058ba4f7a14df6ed9f70d0beac5). Is there another way to specifically pick users from that country?

**Comment by Code4SocialGood** [comment]
@dmgardiner25 Yes, please simple fill in the keyword with country code, and perform a  search to display volunteers or organizations in that country.
